### ISSUE TITLE
Symbol.GetName only works for non-grouped symbol

### ISSUE THREAD

**Issue by dianyancao** [apache/incubator-mxnet#3608]
I use the following code to construct a grouped symbol,got the Error:
symbol.cc:489: Check failed: node == e.source.get() Symbol.GetName only works for non-grouped symbol

```
def get_lenet():
    data = mx.symbol.Variable('data')
    # first conv
    conv1 = mx.symbol.Convolution(data=data, kernel=(5,5), num_filter=20)
    tanh1 = mx.symbol.Activation(data=conv1, act_type="tanh")
    pool1 = mx.symbol.Pooling(data=tanh1, pool_type="max",
                              kernel=(2,2), stride=(2,2))
    # second conv
    conv2 = mx.symbol.Convolution(data=pool1, kernel=(5,5), num_filter=50)
    tanh2 = mx.symbol.Activation(data=conv2, act_type="tanh")
    pool2 = mx.symbol.Pooling(data=tanh2, pool_type="max",
                              kernel=(2,2), stride=(2,2))

    _, out_shapes, _ = pool2.infer_shape(data=(1,1,28,28))
    softmax_class_1 = mx.symbol.Convolution(data=pool2, num_filter=10 + 1,
                                           kernel=(5, 5), stride=(1, 1), pad=(2, 2), name='softmax_class_1')
    _, out_shapes, _ = softmax_class_1.infer_shape(data=(1,1,28,28))
    shape = out_shapes[0][2:]

    softmax_avg_pool = mx.symbol.Pooling(data=softmax_class_1, kernel=(shape[0], 1), stride=(shape[0], 1),
                                         pad=(0, 0), pool_type='avg', name='softmax_avg_pool')

    softmax_class = mx.symbol.SoftmaxActivation(data=softmax_avg_pool, mode='channel', name='softmax_class')
    make_loss = mx.symbol.MakeLoss(data=softmax_class, grad_scale=0.0)

    ctc_label = mx.symbol.Cast(data=mx.symbol.Variable('softmax_label'),dtype='int32')
    ctc_loss = mx.symbol.CTCLoss(data=softmax_class, label=ctc_label, max_label_length=1,
                                name='ctc_class')
    return mx.sym.Group([make_loss, ctc_loss])
```

I want to get the softmax activation to compute metrics,how should I do this?

**Comment by yajiedesign** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to reopen if this is still an active issue. Thanks!
